.azure-pipelines.yml

```diff
@@ -1,6 +1,7 @@
 # https://github.com/actions/virtual-environments
 variables:
-  DEFAULT_NODE_VERSION: "10.x"
+  # v14 fails on Win env: https://github.com/actions/setup-node/issues/411
```

Comment:
have you tried a workaround at https://github.com/actions/setup-node/issues/411#issuecomment-1025543081

I assume we should still use v14 as this is the minimum supported one